packages/react-log-viewer/src/react-window/createListComponent.ts

```diff
@@ -265,6 +269,8 @@ export default function createListComponent({
             ref: innerRef,
             style: {
               height: estimatedTotalSize > height ? estimatedTotalSize : height,
+              /* eslint-disable-next-line no-nested-ternary */
+              width: isTextWrapped ? (hasLineNumbers ? width - 65 : width) : 'auto',
```

Comment:
nit: maybe in your next PR or followup you can save `65` to a variable like `lineNumbersColumnWidth`